M2/Macaulay2/e/interface/polyroots.cpp

```diff
@@ -1,7 +1,9 @@
 #include "interface/factory.h"
 #include "interface/ring.h"
 
+#define register
```

Comment:
What does this do? Perhaps it could benefit from a short comment.

Reply:
Fixed in https://github.com/Macaulay2/M2/pull/1861/commits/7d57cb6e6ea50494e7771a43642b815e02a8569d.